casanovo/config.yaml

```diff
@@ -29,6 +37,31 @@ accelerator: "auto"
 # number will be automatically selected for based on the chosen accelerator.
 devices:
 
+
+###
+# The following parameters are unique to Casanovo's database search mode.
+###
+
+# Enzyme for in silico digestion, used to generate candidate peptides.
+# See pyteomics.parser.expasy_rules for valid enzymes
+enzyme: "trypsin"
+# Digestion type for candidate peptide generation.
+# full: standard digestion. semi: Include products of semi-specific cleavage
+digestion: "full"
+# Number of allowed missed cleavages when digesting protein
+missed_cleavages: 0
+# Maximum number of amino acid modifications per peptide.
+# None generates all possible isoforms as candidates.
+max_mods: 
+# Maximum peptide length to consider
+max_peptide_len: 50
```

Comment:
It would not make sense to include it with `min_peptide_len` as the former influences both denovo and db modes, while `max_peptide_len` only affects db mode. I can see a user being confused as to why Casanovo outputs longer peptides in denovo mode despite specifying a low number in `max_peptide_len`

Reply:
De novo also has a maximum peptide length ([`max_length`](https://github.com/Noble-Lab/casanovo/blob/main/casanovo/config.yaml#L82)), so this needs to be harmonized.

Reply:
Harmonized settings